Formula/gdub.rb

```diff
@@ -6,18 +6,14 @@ class Gdub < Formula
 
   bottle :unneeded
 
-  depends_on "gradle"
+  depends_on "gradle" => :optional
 
   def install
     bin.install "bin/gw"
   end
 
   test do
```

Comment:
Change the test to

```ruby
    assert_match "No gradlew set up for this project", pipe_output("#{bin}/gw 2>&1")
```

We don't really care that much about making sure tests run on people's machines. Working on CI is enough.